### PR TITLE
Bird indicators chapter

### PR DESCRIPTION
**Pull request by ChloeRN**
Submitting the draft of the bird indicators chapter. 

This needs reviewing not just by @anders-kolstad or @DrMattG, but also by @DiegoPavonJordan to ensure that the scripts are doing what they should.  

Like for my other sub-project, I have developed most of the code in a separate repository. Can be found here: https://github.com/NINAnor/birdIndicators

### CODE REVIEW

**Comment by ChloeRN** [comment]
@anders-kolstad can you confirm that you are serious about wanting the final results in a wide format table, i.e. v_YYYY, sd_YYYY etc. should be **COLUMN NAMES** ?
I have implemented it that way but it just goes against everything I believe in 😜

**Comment by ChloeRN** [comment]
@anders-kolstad and @DrMattG please also double-check the situation with the references. I don't think it's supposed to work when rendering single chapters, but please make sure it turns out correctly when rendering the book.

**Review by anders-kolstad — CHANGES_REQUESTED** [review]
See comment about missing data object

**Comment by anders-kolstad** [comment]
Thank you @ChloeRN . I started looking at it now, but got stuck due to a missing file.

Also about the wide format. You could be right. The idea was not to repeat the geometry column in the sf data. In any case you are the only one actually making it to the export stage so far, so I think I will just hash the part of the code that exports the data and also remove any output files from the repo, and then think about output formating later. It should probably go on OSF anyway, and not to GitHub.

**Comment by ChloeRN** [comment]
@anders-kolstad which data object is missing?

File: DRAFT_birdIndicators.Rmd [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+collectSpeciesFiles_Legacy(origin_folder = legacyFile_folder,
+                           target_folder = paste0(folder, "/", subFolderName))
+``` 
+
+#### Lists of species information
+
+The species list files should have been downloaded together with the rest of the `birdIndicators` repository and we can use `makeSpeciesLists()` to assemble the information. 
+
+```{r}
+sppLists <- makeSpeciesLists(Trim_data = Trim_data, spp_DataPath = "temp/birdIndicators-main/data")
```
Comment (anders-kolstad) [inline DRAFT_birdIndicators.Rmd:238]: I cant find _Trim_data_ which is created on line 220 with eval=F but not stored anywhere that I can see.

**Review by anders-kolstad — COMMENTED** [review]
I'm unable to review before I can locate the Trim data

File: DRAFT_birdIndicators.Rmd (hunk 1 above)
Comment (anders-kolstad) [reply DRAFT_birdIndicators.Rmd:238]: There, I added the review again. Not sure what happened...

**Comment by anders-kolstad** [comment]
Removing [eval=F](https://github.com/NINAnor/ecosystemCondition/pull/159/commits/35390937dd104f1fec70e2ec46be8fc3d5715e9b) made new error apprear:

```
Error: nanodbc/nanodbc.cpp:1135: 01S00: [Microsoft][ODBC SQL Server Driver][SQL Server]Cannot open database "TOVTaksering" requested by the login. The login failed.  [Microsoft][ODBC SQL Server Driver]Invalid connection string attribute 
```

This arrises in `DBI::dbConnect()` under `downloadData_TRIM()`

**Comment by anders-kolstad** [comment]
@ChloeRN , I tried again to run `downloadData_TRIM()` from RStudio Server and now I get his error

```
Error: nanodbc/nanodbc.cpp:1125: 00000: [FreeTDS][SQL Server]Login failed for user 'NINA\anders.kolstad'.  [FreeTDS][SQL Server]Cannot open database "TOVTaksering" requested by the login. The login failed.  [FreeTDS][SQL Server]Unable to connect to data source 
```

My used does not have access is seems.

**Comment by ChloeRN** [comment]
I think that means you need to contact Roald Vang and ask for access permission.
Otherwise:
I only use the downloaded data to make the "sppLists" object in the bookdown. I could instead add and read in this file as a .rds in the script. Would that be better?

________________________________

**Comment by ChloeRN** [comment]
Will do this at the start of next week.
________________________________

**Comment by ChloeRN** [comment]
@anders-kolstad I have now added the object `sppLists` as data and changed over to reading that in from folder instead of downloading the entire data from "Hekkefuglovervåkingen" to make it. 
Hopefully that will make the chapter compile for you.

**Review by anders-kolstad — APPROVED** [review]

**Pull request merged**